packages/ses/NEWS.md

```diff
@@ -4,6 +4,25 @@ User-visible changes in SES:
 
 - Adds support for module descriptors better aligned with XS.
 
+- When running Ava tests on Node written in TypeScript, the SES error taming
+  gives line-numbers into the generated JavaScript, in which all compiled code
+  is on line 1. This happens even with the normal development-time lockdown
+  options setting
+  ```js
+  errorTaming: 'unsafe'
+  ```
+  or setting the environment variable
+  ```sh
+  $ export LOCKDOWN_ERROR_TAMING=unsafe
+  ```
+  In addition to the normal `'safe'` and `'unsafe'` settings, this release
+  adds `'unsafe-debug'`. This `errorTaming: 'unsafe-debug'` setting
+  should be used ***during development only*** when you can
+  sacrifice more security for a better debugging experience, as explained at
+  [`errorTaming` Options](https://github.com/endojs/endo/blob/master/packages/ses/docs/lockdown.md#errortaming-options).
+  With this setting, when running Ava tests written in TypeScript on Node,
+  the stacktrace line-numbers should point back into the original
+  TypeScript source, as they would on Node without SES.
```

Comment:
```suggestion
  the stacktrace line-numbers point back into the original
  source, as they do on Node without SES.
```

Reply:
Done. Together, all this reads much better. Thanks!